client/containers/Pub/SpubHeader/PreviewTab/PreviewTab.tsx

```diff
@@ -14,14 +16,15 @@ require('../../PubHeader/draftReleaseButtons.scss');
 type Props = {
 	historyData: any;
 	updateHistoryData: any;
-	pubData: PubPageData;
+	pubData: PubPageData & { submission: DefinitelyHas<Submission, 'submissionWorkflow'> };
```

Comment:
Could we merge this information into `PubPageData`, so that it has:

```ts
submission?: DefinitelyHas<Submission, 'submissionWorkflow'>
```

then the type of `pubData` across the `SpubHeader` and its children would be:

```ts
pubData: DefinitelyHas<PubPageData, 'submission'>;
```

By the way, we have more informative types for `{ historyData, updateHistoryData }` than `any`:

```ts
historyData: PubHistoryState;
updateHistoryData: (patch: Partial<PubHistoryState>) => unknown;
```
I saw you mention elsewhere that using `any` to type these props made you queasy, and it should! Everywhere you see `any` as a prop type it's an artifact of our automatic best-effort JS -> TS conversion from late 2020 — we shouldn't be using it for new props.